src/widgets/CategoryWidget.js

```diff
@@ -24,11 +25,12 @@ import { AggregationTypes } from './AggregationTypes';
   const { column } = props;
   const [categoryData, setCategoryData] = useState(null);
   const [selectedCategories, setSelectedCategories] = useState([]);
-  const [loading, setLoading] = useState(true);
+  const [loading, setLoading] = useState(false);
```

Comment:
Why loading to false? Have in mind the global with SQL

Reply:
You are right, we have a wait gap using global SQL